BUILDING_DETAILS.md

```diff
@@ -5,7 +5,28 @@ This document covers details that may be useful to some users.
 All examples build for `generic-x86-64`. Replace it with
 `generic-armel-iproc` when building for ARM.
 
-## Building additional yocto packages
+## Building additional Yocto packages
+
+Assuming the packages you want to add are already present in one of the
+layers configured in [`bisdn-linux.yaml`](bisdn-linux.yaml), you can build
+them by passing them via `--target` to kas:
+
+```shell
+KAS_MACHINE=generic-x86-64 kas build --target lldpd bisdn-linux.yaml -- --runall=do_package_write_ipk
```

Comment:
```suggestion
KAS_MACHINE=generic-x86-64 kas-container build --target lldpd bisdn-linux.yaml -- --runall=do_package_write_ipk
```

We have a separate section on building without containers, the other examples should use `kas-container`. I realize that I accidentally introduced an example with `kas` as well, I can fix that or you can add it to this PR, too.